boot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Trajectory Controller  (not together with the position controllers)
  joint_trajectory_controller:
    type: effort_controllers/JointTrajectoryController
    # which joints to control:
    joints:
      - knee
      - hip2
      - back_lh
      - shoulder_l
      - shoulder_psi_l
      - shoulder_theta_l
      - elbow_theta_l
      - elbow_xx_l
      - elbow_left
      
    constraints:
      goal_time: 0.5
      stopped_velocity_tolerance: 0.01
      knee                   : {trajectory: 0.01, goal: 0.01}
      hip2                   : {trajectory: 0.01, goal: 0.01}
      back_lh                : {trajectory: 0.01, goal: 0.01}
      shoulder_l             : {trajectory: 0.01, goal: 0.01}
      shoulder_psi_l         : {trajectory: 0.01, goal: 0.01}
      shoulder_theta_l       : {trajectory: 0.01, goal: 0.01}
      elbow_theta_l          : {trajectory: 0.01, goal: 0.01}
      elbow_xx_l             : {trajectory: 0.01, goal: 0.01}
      elbow_left             : {trajectory: 0.01, goal: 0.01}
      
    gains:
      knee                   :  {p: 100.0, d: 1.0, i: 0.0, i_clamp: 0.0}
      hip2                   :  {p: 500.0, d: 1.0, i: 0.0, i_clamp: 0.0}
      back_lh                :  {p: 100.0, d: 1.0, i: 0.0, i_clamp: 0.0}
      shoulder_l             :  {p: 100.0, d: 1.0, i: 0.0, i_clamp: 0.0}
      shoulder_psi_l         :  {p: 100.0, d: 1.0, i: 0.0, i_clamp: 0.0}
      shoulder_theta_l       :  {p: 100.0, d: 1.0, i: 0.0, i_clamp: 0.0}
      elbow_theta_l          :  {p: 100.0, d: 1.0, i: 0.0, i_clamp: 0.0}
      elbow_xx_l             :  {p: 100.0, d: 1.0, i: 0.0, i_clamp: 0.0}
      elbow_left             :  {p: 100.0, d: 1.0, i: 0.0, i_clamp: 0.0}

      state_publish_rate:  50       # was 50. stabieler?
      stop_trajectory_duration: 0.0

